src/QMCWaveFunctions/Jastrow/RPAJastrow.cpp

```diff
@@ -117,19 +113,19 @@ void RPAJastrow::buildOrbital(const std::string& name,
   app_log() << "    RPAJastrowBuilder::addTwoBodyPart Rs = " << Rs << "  Kc= " << Kc << std::endl;
   if (rpafunc == "yukawa" || rpafunc == "breakup")
   {
-    myHandler = new LRHandlerTemp<YukawaBreakup<RealType>, LPQHIBasis>(targetPtcl, Kc);
+    myHandler = std::make_unique<LRHandlerTemp<YukawaBreakup<RealType>, LPQHIBasis>>(targetPtcl, Kc);
   }
   else if (rpafunc == "rpa")
   {
-    myHandler = new LRRPAHandlerTemp<RPABreakup<RealType>, LPQHIBasis>(targetPtcl, Kc);
+    myHandler = std::make_unique<LRRPAHandlerTemp<RPABreakup<RealType>, LPQHIBasis>>(targetPtcl, Kc);
   }
   else if (rpafunc == "dyukawa")
   {
-    myHandler = new LRHandlerTemp<DerivYukawaBreakup<RealType>, LPQHIBasis>(targetPtcl, Kc);
+    myHandler = std::make_unique<LRHandlerTemp<DerivYukawaBreakup<RealType>, LPQHIBasis>>(targetPtcl, Kc);
   }
   else if (rpafunc == "drpa")
   {
-    myHandler = new LRRPAHandlerTemp<DerivRPABreakup<RealType>, LPQHIBasis>(targetPtcl, Kc);
+    myHandler = std::make_unique<LRRPAHandlerTemp<DerivRPABreakup<RealType>, LPQHIBasis>>(targetPtcl, Kc);
   }
   else
   {
```

Comment:
Remove all the {} around one line and replace APP_ABORT with exception in the next line.